File: C/Code/10800/10818 - 최소, 최대.c
#include <stdio.h>

int main(n) {
	int max = -9999999, min = 9999999;
	for (scanf("%d", &n); n; n--) {
		int temp;
		scanf("%d", &temp);
		max = (temp > max) ? temp : max;
		min = (temp < min) ? temp : min;
	}
	printf("%d %d", min, max);
	return 0;
}

File: Java/Code/10800/10818 - 최소, 최대.java
import java.util.Scanner;

public class Main {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int max = -1000001, min = 1000001;
		for (int t = sc.nextInt(); t > 0; t--) {
			int a = sc.nextInt();
			if (a < min)
				min = a;
			if (a > max)
				max = a;
		}
		System.out.print(min + " ");
		System.out.print(max);
	}
}